From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Fri, 16 Oct 2020 20:27:27 +0000
Subject: [PATCH] Remove requirement on tensorboard-plugin-wit

This dependency appears to be optional, since it's a plugin. If a user
wants this plugin, they can run `pip install tensorboard-plugin-wit`.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 tensorboard/pip_package/requirements.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tensorboard/pip_package/requirements.txt b/tensorboard/pip_package/requirements.txt
index de7f134..647b6e0 100644
--- a/tensorboard/pip_package/requirements.txt
+++ b/tensorboard/pip_package/requirements.txt
@@ -27,7 +27,6 @@ protobuf >= 3.6.0
 requests >= 2.21.0, < 3
 setuptools >= 41.0.0
 six >= 1.10.0
-tensorboard-plugin-wit >= 1.6.0
 werkzeug >= 0.11.15
 # python3 specifically requires wheel 0.26
 wheel; python_version < "3"
